services/reporting/reporting.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+package reporting
+
+import (
+	"time"
+
+	"github.com/getsentry/sentry-go"
+)
+
+// Return the options that can be configured for the error reporting
+// library.
+func Options() sentry.ClientOptions {
+	return sentry.ClientOptions{}
+}
+
+// Initialize the reporting services.
+func Init(options sentry.ClientOptions) error {
+	return sentry.Init(options)
+}
+
+// Capture an error and report it.
+func CaptureException(err error) *sentry.EventID {
```

Comment:
I would prefer to not leak the sentry types here, I would prefer here to return always, in this case, a `*string`

Reply:
Yes, that would be desirable, thing is that the `sentry.ClientOptions` type is already leaking, and hide that one is something that I don't know if it's worth it. What do you think?